datalad/interface/tests/test_crawl_init.py

```diff
@@ -0,0 +1,66 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+
+from nose.tools import eq_, assert_raises
+from ...api import crawl_init
+from os import remove
+from os.path import exists
+from datalad.support.annexrepo import AnnexRepo
+from datalad.tests.utils import with_tempfile, chpwd
+from datalad.consts import CRAWLER_META_CONFIG_PATH, CRAWLER_META_DIR
+
+
+@with_tempfile(mkdir=True)
+def test_crawl_init_openfmri_args(tmpdir):
+    ar = AnnexRepo(tmpdir)
+    with chpwd(tmpdir):
+        crawl_init(args={'dataset': 'ds000001'}, template='openfmri')
+        eq_(exists(CRAWLER_META_DIR), True)
+        eq_(exists(CRAWLER_META_CONFIG_PATH), True)
+        f = open(CRAWLER_META_CONFIG_PATH, 'r')
+        contents = f.read()
+        eq_(contents, '[crawl:pipeline]\ntemplate = openfmri\n_dataset = ds000001\n\n')
+
+
+@with_tempfile(mkdir=True)
+def test_crawl_init_openfmri(tmpdir):
+    ar = AnnexRepo(tmpdir)
+    with chpwd(tmpdir):
+        crawl_init(template='openfmri', template_func='superdataset_pipeline')
+        eq_(exists(CRAWLER_META_DIR), True)
+        eq_(exists(CRAWLER_META_CONFIG_PATH), True)
+        f = open(CRAWLER_META_CONFIG_PATH, 'r')
+        contents = f.read()
+        eq_(contents, '[crawl:pipeline]\ntemplate = openfmri\nfunc = superdataset_pipeline\n\n')
+
+
+@with_tempfile(mkdir=True)
+def test_crawl_init_fcp(tmpdir):
+    ar = AnnexRepo(tmpdir)
+    with chpwd(tmpdir):
+        crawl_init(args=['dataset=Baltimore', 'tarballs=True'], template='fcptable')
+        eq_(exists(CRAWLER_META_DIR), True)
+        eq_(exists(CRAWLER_META_CONFIG_PATH), True)
+        f = open(CRAWLER_META_CONFIG_PATH, 'r')
+        contents = f.read()
+        eq_(contents, '[crawl:pipeline]\ntemplate = fcptable\n_dataset = Baltimore\n_tarballs = True\n\n')
```

Comment:
now try to generalize your 3 nearly identical tests into 1 parametric.... see e.g. those which you find with `git grep 'yield _test'`